box/example_test.go

```diff
@@ -0,0 +1,43 @@
+// Run Tarantool Common Edition before example execution:
+//
+// Terminal 1:
+// $ cd box
+// $ TEST_TNT_LISTEN=127.0.0.1:3013 tarantool testdata/config.lua
+//
+// Terminal 2:
+// $ go test -v example_test.go
+package box_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/tarantool/go-tarantool/v2/box"
+)
+
+func Example() {
```

Comment:
Please, add a separate example for the Box.Info() too.

Reply:
i add it to one example, now there are 2 cases how to get info with equal uuid log

Reply:
Please, let's it better be 2 examples.